pkg/reconciler/pipelinerun/pipelinerun.go

```diff
@@ -817,6 +818,13 @@ func combineTaskRunAndTaskSpecAnnotations(pr *v1beta1.PipelineRun, pipelineTask
 	return annotations
 }
 
+func getPipelineRunTimeout(pr *v1beta1.PipelineRun) metav1.Duration {
+	if pr.Spec.Timeout == nil {
+		return metav1.Duration{Duration: config.DefaultTimeoutMinutes * time.Minute}
```

Comment:
Is `config.FromContextOrDefaults(ctx).DefaultTimeoutMinutes` the newer way to get the value in case it's been overridden?  I see that used in the taskrun reconciler.  Not sure if applicable here as well.

Reply:
oh right, I can work with that :+1: